src/ReactRotatingText.js

```diff
@@ -89,12 +90,17 @@ class ReactRotatingText extends React.Component {
 
   _animate() {
     const { index } = this.state;
-    const { items, pause, emptyPause, eraseMode } = this.props;
+    const { items, pause, emptyPause, eraseMode, random } = this.props;
     const type = this._type;
     const erase = this._erase;
     const overwrite = this._overwrite;
     const loopingFunc = this._animate.bind(this);
-    const nextIndex = index === items.length - 1 ? 0 : index + 1;
+    var nextIndex;
```

Comment:
Can you change this to use `let` instead?

Reply:
done